test/loader/test_link_neighbor_loader.py

```diff
@@ -204,6 +204,33 @@ def test_link_neighbor_loader_edge_label():
         assert torch.all(batch.edge_label[10:] == 0)
 
 
+@withPackage("pyg_lib")
+def test_temporal_homo_link_neighbor_loader():
+    data = Data(
+        x=torch.randn(10, 5),
+        edge_index=torch.randint(0, 10, (2, 1234)),
+        edge_time=torch.arange(1234),
+    )
+
+    batch_size = 1
+    loader = LinkNeighborLoader(
+        data,
+        num_neighbors=[-1],
+        time_attr="edge_time",
+        edge_label_time=data.edge_time,
+        edge_label=torch.ones(data.num_edges),
+        batch_size=batch_size,
+        shuffle=True,
+    )
+
+    for sample in loader:
+        assert sample.edge_label_index.size() == (2, batch_size)
+        assert sample.edge_label_time.size() == (batch_size, )
+        assert sample.edge_label.size() == (batch_size, )
+        # TODO(akihironitta): Investigate the failure
+        # assert torch.all(sample.edge_time <= sample.edge_label_time)
```

Comment:
Could someone also confirm this assertion is correct before I dig into this?

Reply:
Yes, will do.